dev-python/statsmodels/statsmodels-0.10.2.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Statistical computations and models for use with SciPy"
+HOMEPAGE="https://www.statsmodels.org/stable/index.html"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples test"
+
+COMMON_DEPEND="
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/pandas-0.23.0[${PYTHON_USEDEP}]
+	dev-python/patsy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+"
+RDEPEND="${COMMON_DEPEND}"
+DEPEND="${COMMON_DEPEND}
+	dev-python/cython[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/ipykernel[${PYTHON_USEDEP}]
+		dev-python/jupyter_client[${PYTHON_USEDEP}]
+		dev-python/matplotlib[${PYTHON_USEDEP}]
+		dev-python/nbconvert[${PYTHON_USEDEP}]
+		dev-python/nbformat[${PYTHON_USEDEP}]
+		dev-python/numpydoc[${PYTHON_USEDEP}]
+		dev-python/sphinx[${PYTHON_USEDEP}]
```

Comment:
I doubt you need PYTHON_USEDEP here...

Reply:
most ebuilds under `dev-python` have sphinx as `dev-python/sphinx[${PYTHON_USEDEP}]`

Reply:
The easiest thing to do nowadays is to rely on [`distutils_enable_sphinx`](https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#distutils_enable_sphinx).

.

Reply:
^ Please try to use this. Why `dev-python/sphinx[${PYTHON_USEDEP}]` is common because everyone's just accustomed to writing `PYTHON_USEDEP` after any `dev-python/*` package, but with sphinx it'd be wrong in most cases.